UAT/V1.10.6_patch.sql

```diff
@@ -0,0 +1,11 @@
+UPDATE 
+	cqc."ServicesCapacity"
+SET
+	"Type"='Utilisation'
+WHERE
+	"ServiceID" in (11, 13, 18, 20, 22, 35);
```

Comment:
technically correct yes. 20, 13, and 11 are already `utilisations`. :-)

Reply:
fixed.